.nvmrc

```diff
@@ -1 +1 @@
->=12.22
```

Comment:
how about `16.16.0`

Reply:
I have no opinion on this, all fine to me, personally. So I'll adopt this quickly.